public/site.webmanifest

```diff
@@ -6,14 +6,9 @@
             "src": "/android-chrome-192x192.png",
             "sizes": "192x192",
             "type": "image/png"
-        },
-        {
-            "src": "/android-chrome-384x384.png",
-            "sizes": "384x384",
-            "type": "image/png"
         }
     ],
-    "theme_color": "#000000",
-    "background_color": "#000000",
+    "theme_color": "#ffffff",
+    "background_color": "#ffffff",
```

Comment:
not sure we want to update that part

Reply:
Sorry about the late respond, I will correct them